project_one.html

```diff
@@ -204,7 +290,9 @@ <h2>Contact me</h2>
         placeholder="Write your message here"></textarea>
       <button type="submit" class="button">Get It touch</button> 
       </form>
```

Comment:
- [ ] I noticed that your buttons do not have the `active` and `disabled` state. Please ensure to include that in your design implementation